### PR TITLE
Move filter to above the card list

### PR DESCRIPTION
**Pull request by rosejbon**
Issue: https://trello.com/c/m2vfAe2I/143-move-the-filter-to-above-the-list-of-results-vvv-e

### CODE REVIEW

File: src/Components/CardList.tsx [hunk 1]
```diff
@@ -297,10 +298,10 @@ export const CardList = ({
             totalPages={totalPages}
             isFirstPage={isFirstPage}
             isLastPage={isLastPage}
-            showFilterBtn
```
Comment (Nikomus) [inline src/Components/CardList.tsx:300]: If we never want to pass this prop into CardListNavigation, we should clean up and remove what it controls in that component.

File: src/Components/CardList.tsx [hunk 2]
```diff
@@ -297,10 +298,10 @@ export const CardList = ({
...
           />
           <List>
             {listType === 'filtered' ? <QuizEncouragement /> : null}
+            <FilterButton />
```
Comment (Nikomus) [inline src/Components/CardList.tsx:304]: I think the filter button should only be displayed on the "All" page - it's the only one that it acts on. You an use the line above to conditionally render it on that page only.

**Review by edpe — COMMENTED** [review]
Nice button, I like the combination of the icon and the text :hamster:

File: src/Components/CardList.tsx (hunk 2 above)
Comment (rosejbon) [reply src/Components/CardList.tsx]: Nice :D I've updated that

File: src/Components/CardList.tsx (hunk 1 above)
Comment (rosejbon) [reply src/Components/CardList.tsx:300]: Done :D

**Review by Nikomus — APPROVED** [review]
Looks great! 🌹

**Pull request merged**